templates/frameworks/list.html.twig

```diff
@@ -113,98 +111,45 @@
-
-                    {% if pagination.totalResults == 1 %}
-                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
-                            framework found</h2>
+                {% if guided_match_flag is defined %}
+                    {% if query is defined and query in keywords%}
+                    <div class="govuk-grid-column-two-thirds guided-match-background govuk-!-margin-bottom-2 guided-match-area">
+                        {% if pagination.totalResults == 1 %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }} framework found</h2>
+                        {% else %}
+                            <h2 class="govuk-heading-l ccs-font-weight-semibold text-white govuk-!-margin-1">Get help finding the right framework</h2>
+                            <h2 class="govuk-heading-s ccs-font-weight-semibold text-white">There are "{{ pagination.totalResults }}" results found for {{ query }}. Depending on your requirements, some of these frameworks maybe better suited than others.</h2>
+                        {% endif %}
+
+                        <a href={{ match_url }} role="button" draggable="false" class="govuk-button govuk-button--start govuk-!-margin-bottom-3" data-module="govuk-button">
+                            Start now
+                            <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+                            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+                            </svg>
+                        </a>
+                    </div>
                     {% else %}
-                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
-                            frameworks found</h2>
+                    <div class="govuk-grid-column-two-thirds">
+
+                        {% if pagination.totalResults == 1 %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
+                                framework found</h2>
+                        {% else %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
+                                frameworks found</h2>
+                        {% endif %}
+                    
+                    </div>
                     {% endif %}
-
-
+                {% else %}
+                    <div class="govuk-grid-column-two-thirds">
+                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }} framework found</h2>
+                    </div>
+                {% endif %}
```

Comment:
This is the else case for `{% if guided_match_flag is defined %}` which does not seem right.

Reply:
The else case is if the flag isn't defined aka when its input in the query is missing therefore it shows the normal screen. This is the toggled off state of the feature flagging.